### ISSUE TITLE
tests: testCopyObjectV2 should initialize a V2 client instead of V4. 

### ISSUE THREAD

**Issue by harshavardhana** [minio/minio-go#754]
```go
// Tests copy object
func testCopyObjectV2() {
        logTrace()

        // Seed random based on current time.
        rand.Seed(time.Now().Unix())

        // Instantiate new minio client object
        c, err := minio.NewV4(
                os.Getenv(serverEndpoint),
                os.Getenv(accessKey),
                os.Getenv(secretKey),
                mustParseBool(os.Getenv(enableHTTPS)),
        )
        if err != nil {
                log.Fatal("Error:", err)
        }
```

**Comment by poornas** [comment]
fixed in #757. @deekoder, this can be closed

**Comment by harshavardhana** [comment]
Closing as fixed.
